src/components/ui/widgets/FgTooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import { Tooltip, Typography } from '@material-tailwind/react';
+
+type FgTooltipProps = {
+  as?: React.ElementType;
+  variant?: 'outline' | 'ghost';
+  link?: string;
+  disabledCondition?: boolean;
+  onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  icon?: React.ElementType;
+  label: string;
+  triggerClasses?: string;
+  openCondition?: boolean;
+  children?: React.ReactNode;
+};
+
+export default function FgTooltip({
+  as,
+  variant,
+  link,
+  disabledCondition,
+  onClick,
+  icon,
+  label,
+  triggerClasses,
+  openCondition,
+  children
+}: FgTooltipProps) {
+  const Component = as || null;
+  const Icon = icon || null;
+
+  return (
+    <Tooltip placement="top" open={openCondition}>
```

Comment:
I am assuming you haven't need to place a tooltip in any other placement up to this point. Since that is the case, setting top is fine, but the first time we need to place the tooltip in a different placement, we should make this an option with top as the default.